lib/danger/danger_core/plugins/dangerfile_gitlab_plugin.rb

```diff
@@ -91,8 +159,40 @@ def api
       @gitlab.client
     end
 
+    # @!group GitLab Misc
+    # Returns a list of HTML anchors for a file, or files in the head repository. An example would be:
+    # `<a href='https://gitlab.com/artsy/eigen/blob/561827e46167077b5e53515b4b7349b8ae04610b/file.txt'>file.txt</a>`. It returns a string of multiple anchors if passed an array.
+    # @param    [String or Array<String>] paths
+    #           A list of strings to convert to gitlab anchors
+    # @param    [Bool] full_path
+    #           Shows the full path as the link's text, defaults to `true`.
+    #
+    # @return [String]
+    def html_link(paths, full_path: true)
```

Comment:
Hrm, perhaps perfect is the enemy of done, we can check if it's the same repo and link it using relative urls? Otherwise display something without a link